src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(Color color, double base1, double base2, double height) {
+        super(String.valueOf(color));
```

Comment:
we should pass String to constructor.
Also, to get string representation of enum use .name()